content/src/service/synchronization/EventDeployer.ts

```diff
@@ -185,7 +185,15 @@ export class EventDeployer {
     const deploymentExecution = await deploymentPreparation
     return async () => {
       try {
-        await deploymentExecution.execution()
+        const response: Timestamp | ErrorList = await deploymentExecution.execution()
+        if (typeof response != 'number') {
```

Comment:
I'm not loving all the `typeof response != 'number'`. Let's create a helper/utils that given a deployment results, returns whether it was successful or not